### DISCUSSION TITLE
退款（已交费未能成功续期2年专业版 ，已过近3个月账户还是免费版）

### DISCUSSION THREAD

**Discussion by ArvieWang** [getlantern/lantern-forum-cn#1042] [General]
![380476460-9006a72f-445f-4cf7-afc9-cac75d20fa39](https://github.com/user-attachments/assets/54e02d4d-4f96-48b7-8515-8f1f414f37b6)
如上图所示，于2024年9月24日充值2年蓝灯专业版，但是一直提示即将到期，未成功充值，
邮箱：[359264293@qq.com](mailto:359264293@qq.com)
订单号：2024092422001467311438056063
商家订单号：2024092409411995523

我之前讨论的帖子https://github.com/getlantern/lantern-forum-cn/discussions/1000
一直没处理，不能成功到账，那就退款处理！！！！！！！

**Comment by ArvieWang** [comment]
@Mike777Khoo

**Comment by MaxMKMax** [comment]
@ArvieWang 您好. 已经帮您提交问题上去了. 非常抱歉啊. 谢谢您的反馈.

**Reply by ArvieWang** [reply]
@Mike777Khoo  何时能退款到账？
